### ISSUE TITLE
Bloop will not clone submodules when cloning a new repo

### ISSUE THREAD

**Issue by yqs112358** [BloopAI/bloop#1130]
**Describe the bug**
When cloning a new repo, Bloop will not clone submodules, only the target repo itself, which may cause the code to be incomplete .

**Expected behavior**
Execute `git clone --recurse-submodules` instead of simple `git clone` to get the content of all submodules in repo.

**To Reproduce**
How can we reproduce the bug? Helpful information could include:
- Find any repo with submodules, such as `https://github.com/Krafalski/git-submodule-test`
- Try to clone it in Bloop app
- You will get a repo without the submodule content

**Screenshots or output**
<img width="410" alt="image" src="https://github.com/BloopAI/bloop/assets/37969157/06857d67-493e-4c63-8cb9-4ca3476a4d94">

![image](https://github.com/BloopAI/bloop/assets/37969157/7cf46f0e-7087-4fb2-8750-4df99ce9dd4d)

**Comment by rsdy** [comment]
This is correct. We're working on this feature, and it should land in a Bloop release in the not too distant future!

**Comment by yqs112358** [comment]
Okay. Looking forward to this new feature😘
